core/src/main/java/org/jobrunr/server/BackgroundJobPerformer.java

```diff
@@ -115,6 +115,12 @@ private void updateJobStateToFailedAndRunJobFilters(String message, Exception e)
         } catch (Exception badException) {
             LOGGER.error("ERROR - could not update job(id={}, jobName='{}') to FAILED state", job.getId(), job.getJobName(), badException);
         }
+
```

Comment:
Is there a reason you put it here? In my eyes, this block should be in the `try` statement, right before the `catch` clause.

Reply:
Thanks for your review. It depends on whether or not the job exception need to be logged if an error occurs in the `try` block.

I'm ok to move the log to right before the `catch` clause if you prefer not to log the job exception.

Reply:
Yeah, I do prefer that from a code style perspective.

This other 2 methods are JobRunr internal and normally don't fail (except e.g. when storage is down) in which case we could not update the job anyway and it will be retried.

Can you move it? Then I'll approve it and create a new release...

Thanks for all your hard work!

Reply:
No worries. Pushed the updates.